api/tools/src/main/java/org/pmiops/workbench/tools/cdrConfig/test/README.md

```diff
@@ -0,0 +1,29 @@
+# Test files for UpdateCdrConfig
```

Comment:
Looking at our existing source folders, I'd probably call the package `cdrconfig` without the camelcasing

Reply:
sure